applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -82,6 +82,23 @@ static const struct state_switch state_switch[] = {
 #endif /* CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START */
 };
 
+enum ble_bond_opt {
+	BLE_BOND_OPT_MODULE_NAME,
```

Comment:
I think we should make a generic way of passing module name (keep it in macro). One can easily put the list in the wrong order.

Reply:
done